src/compiler/typescript_to_rust/generators/azle_into_js_value_trait.ts

```diff
@@ -64,6 +64,83 @@ export function generateAzleIntoJsValueTrait(): Rust {
             }
         }
 
+        impl AzleIntoJsValue for ic_cdk::api::call::RejectionCode {
+            fn azle_into_js_value(self, context: &mut boa_engine::Context) -> boa_engine::JsValue {
+                match self {
+                    ic_cdk::api::call::RejectionCode::NoError => {
+                        boa_engine::object::ObjectInitializer::new(context)
+                            .property(
+                                "NoError",
+                                boa_engine::JsValue::Null,
+                                boa_engine::property::Attribute::all(),
+                            )
+                            .build()
+                            .into()
+                    }
```

Comment:
I don't understand the syntax here, I thought a comma would be necessary

Reply:
Commas between `.build()` and `.into()`? This is a boa ObjectInitializer. From what I'm seeing in [the docs](https://docs.rs/boa_engine/0.15.0/boa_engine/object/struct.ObjectInitializer.html) and other places in the codebase you call `::new` on it to generate a new builder thing, then call `.property` to add properties, and finish it up with a `.build` to convert it from an ObjectInitializer to an Object. The `.into` just makes it a JsValue. So, commas within calls to `.property` but not between the other calls.

Happy to talk in person if I'm just not understanding the question or something.